invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/RequestLoanModal/RequestLoanModal.js

```diff
@@ -0,0 +1,181 @@
+import React, { Component } from 'react';
+import {
+  Button,
+  Header,
+  Icon,
+  Modal,
+  Segment,
+  Grid,
+  Label,
+  Input,
+  Divider,
+} from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import { PatronSearchBox } from '../../../../../common/components/PatronSearchBox';
+
+export default class RequestLoanModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      open: false,
+      currentSelection: null,
+      disabledSearch: false,
+      selectedDeliveryMethod: null,
+    };
+    this.requestNewLoanForDocument = this.props.requestNewLoanForDocument.bind(
+      this
+    );
+  }
+
+  toggle = () => {
+    this.setState({
+      open: !this.state.open,
+      currentSelection: null,
+      disabledSearch: false,
+    });
+  };
+
+  handleDeliveryMethodChange = e => {
+    this.setState({ selectedDeliveryMethod: e.target.value });
+  };
+
+  requestLoanButton = (
+    <Button
+      positive
+      icon="add"
+      labelPosition="left"
+      content="Request Loan"
+      onClick={this.toggle}
+    />
+  );
+
+  deliveryMethodInput = (
+    <Input
+      action={{
+        color: 'teal',
+        labelPosition: 'left',
+        icon: 'truck',
+        content: 'Delivery method',
```

Comment:
Delivery method is a CERN thing, not for a generic ILS.
Please remove it.
Can you also double check that there is nothing about that in the frontend/backend source code?